File: VisualBasic/GlobalVariables.vb
Public Class GlobalVariables
    Public Shared ServerID As Integer = My.Settings.LastProcessID
    Public Shared ServerPath As String = System.Text.RegularExpressions.Regex.Match(Trim(My.Settings.ServerPath), "^(.*[^\\])").Value
    Public Shared ServerEXE As String = Trim(My.Settings.ServerEXE)
    Public Shared ProcessName As String = GlobalFunctions.GetProcessName(ServerEXE)
    Public Shared ServerArgs As String = Trim(My.Settings.ServerStartArguments)
    Public Shared ServerStopCmd As String = Trim(My.Settings.ServerStopString)
    Public Shared AutoStart As Boolean = My.Settings.AutoStart
End Class

File: CSharp/GlobalVariables.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp
{
    public class GlobalVariables
    {
        public static int ServerID = Properties.Settings.Default.LastProcessID;
        public static string ServerPath = System.Text.RegularExpressions.Regex.Match(Properties.Settings.Default.ServerPath.Trim(), @"^(.*[^\\])").Value;
        public static string ServerEXE = Properties.Settings.Default.ServerEXE.Trim();
        public static string ProcessName = GlobalFunctions.GetProcessName(ServerEXE);
        public static string ServerArgs = Properties.Settings.Default.ServerStartArguments.Trim();
        public static string ServerStopCmd = Properties.Settings.Default.ServerStopString.Trim();
        public static bool AutoStart = Properties.Settings.Default.AutoStart;
    }

}
